### PR TITLE
Add napari-indices recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
Haven't heard from the author in some time. The author can be added as maintainer in a subsequent PR.
        
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/napari-indices/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - napari_indices
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+    - pyqt
+
+about:
+  home: https://github.com/Emmanulla0/napari-indices
```
Comment (jaimergp) [inline recipes/napari-indices/meta.yaml:41]:
```suggestion
  home: https://pypi.org/project/napari-indices/
```

Github repo is 404.

File: recipes/napari-indices/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - magicgui
+    - matplotlib
+    - napari
+    - numpy
+    - python >=3.8
+    - spectral
```
Comment (jaimergp) [inline recipes/napari-indices/meta.yaml:27]:
```suggestion
    - spectral
    - tifffile
```

File: recipes/napari-indices/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{% set name = "napari-indices" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-indices-{{ version }}.tar.gz
```
Comment (jaimergp) [inline recipes/napari-indices/meta.yaml:9]: Project imports PyQt5 directly.

**Review by jaimergp — CHANGES_REQUESTED** [review]
Project imports PyQt5 directly. Needs to be patched.

**Comment by goanpeca** [comment]
>Project imports PyQt5 directly. Needs to be patched.

Fixed 👍🏼

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

**Review by xhochy — APPROVED** [review]
Requested changes were addressed.

**Pull request merged**